crates/js-sys/src/lib.rs

```diff
@@ -1154,6 +1154,12 @@ extern "C" {
     #[wasm_bindgen(static_method_of = Math)]
     pub fn fround(x: f64) -> f32;
 
+    /// The Math.hypot() function returns the square root of the sum of squares of its arguments.
+    ///
+    /// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/fround
```

Comment:
Shouldn't this be https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/hypot ?

Reply:
Ah yes indeed! @jontro want to file a follow-up PR?

Reply:
Done! https://github.com/rustwasm/wasm-bindgen/pull/567